app/controllers/flowcommerce_spree/webhooks_controller.rb

```diff
@@ -2,14 +2,17 @@
 
 module FlowcommerceSpree
   class WebhooksController < ActionController::Base
+    wrap_parameters false
     respond_to :json
 
     # forward all incoming requests to Flow WebhookService object
     # /flow/event-target endpoint
     def handle_flow_web_hook_event
-      webhook_result = WebhookService.process(params[:webhook])
-      result = {}
-      result[:error] = webhook_result.full_messages.join("\n") if webhook_result.errors.any?
+      result = check_organization
```

Comment:
The `check_organization` is just to ensure the data is coming for the correct environment - the organizations are mejuridev or mejuri-staging, etc.

Also, we're not raising an error if the result is blank - we're going further with WebhookService.process. I prefer to not use exceptions where we could do without them, because the exceptions are slow.